lib/Reconciler.lua

```diff
@@ -264,10 +241,6 @@ function Reconciler._mountInternal(element, parent, key, context)
 			_children = children,
 			_rbx = target,
 		}
-	elseif typeof(element) == "boolean" then
-		-- Ignore booleans of either value
-		-- See https://github.com/Roblox/roact/issues/14
-		return nil
```

Comment:
We should keep this check inside this function. If we're going to move it, let's put it first so that we can avoid indexing into a bool/nil element.